doc/doxygen/src/getting-started.md

```diff
@@ -32,8 +32,94 @@ information that can help you with your platform:
 
-The build system                                            {#the-build-system}
-----------------
+Building and executing an example        {#building-and-executing-an-example}
+---------------------------------
+RIOT provides a number of examples in the `examples/` directory. Every example
+has a README that documents its usage and its purpose. You can build them by
+typing
+
+~~~~~~~~ {.sh}
+make BOARD=samr21-xpro
+~~~~~~~~
+
+or
+
+~~~~~~~~ {.sh}
+make all BOARD=samr21-xpro
+~~~~~~~~
+
+into your shell.
+
+To flash the application to a board just type
+
+~~~~~~~~ {.sh}
+make flash BOARD=samr21-xpro
+~~~~~~~~
+
+You can then access the board via the serial interface:
+
+~~~~~~~~ {.sh}
+make term BOARD=samr21-xpro
+~~~~~~~~
+
+If you are using multiple boards you can use the `PORT` macro to specify the
+serial interface:
+
+~~~~~~~~ {.sh}
+make term BOARD=samr21-xpro PORT=/dev/ttyACM1
+~~~~~~~~
+
+Note that the `PORT` macro has a slightly different semantic in `native`. Here
+it is used to provide the name of the TAP interface you want to use for the
+virtualized networking capabilities of RIOT.
+
+We use `pyterm` as the default terminal application. It is shipped with RIOT in
+the `dist/tools/pyterm/` directory. If you choose to use another terminal
+program you can set `TERMPROG` (and if need be the `TERMFLAGS`) macros:
+
+~~~~~~~~ {.sh}
+make -C examples/gnrc_networking/ term \
+    BOARD=samr21-xpro \
+    TERMPROG=gtkterm \
+    TERMFLAGS="-s 115200 -p /dev/ttyACM0 -e"
+~~~~~~~~
+
+Using the native port with networking  {#using-the-native-port-with-networking}
+-------------------------------------
+RIOT provides a special [native port][native] with which you can run the
+complete RIOT stack in a process on your *NIX system. To enable networking
+between multiple native RIOT instances, you need to use tun/tap interfaces.
+
+First, you need to make sure that you're compiling RIOT for the native port
+and add tun/tap support. You can do this by compiling your project with the following environment variables:
+
+    BOARD=native USEMODULE+=netdev_tap make
+
+> **Note:** if you're using RIOTs default network stack, you should add
+> `gnrc_netdev_default` and `auto_init_gnrc_netif` instead, which have the
+> `netdev_tap` module included.
+
+Now, you can connect several native RIOT instances to each other or enable
+communication between a native RIOT instance and its (*NIX) host system.
+
+To connect multiple native RIOT instances to each other, use the shell script
+called `tapsetup` in `RIOT/dist/tools/tapsetup`.
+First, create a bridge and two (or `count` at your option) tap interfaces:
+
+     ./dist/tools/tapsetup/tapsetup [-c [<count>]]
+
+And follow the instructions of the script telling you to start RIOT instances
+on specific tap interfaces. To start a RIOT instance on a specific tap
+interface, run
+
+    BOARD=native USEMODULE+=netdev_tap PORT=<tap interface name> make term
```

Comment:
I guess this was your reaction to https://github.com/RIOT-OS/RIOT/pull/10256#discussion_r231580923 .That was not was I meant. [The instruction assumed](https://github.com/RIOT-OS/RIOT/pull/10256/files#diff-0d059750c8f90e4bb682af7b582ec164R96) so far, that the `netdev_tap` module is included. If that is the case `PORT` will never be added to the `TERMFLAGS`, so not provided to the application:

https://github.com/RIOT-OS/RIOT/blob/6e9612f1d1e41ef56b79c1fcddc3d7e3aae2b9d7/boards/native/Makefile.include#L72-L79